test/e2e/storage-endpoint.integration.test.js

```diff
@@ -1,12 +1,57 @@
 const supertest = require("supertest");
-const { put } = require("../../src/storage/s3");
+const { put, remove } = require("../../src/storage/s3");
 const config = require("../../src/storage/config");
 const ropstenDocument = require("../fixtures/certificate.json");
 
 const API_ENDPOINT = "https://api-ropsten.opencerts.io";
 const request = supertest(API_ENDPOINT);
 
+const uuidV4Regex = new RegExp(
+  /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i
+);
+
+const thatIsUploadResponse = {
+  id: expect.stringMatching(uuidV4Regex),
+  key: expect.any(String),
+  type: expect.stringMatching("OPEN-ATTESTATION-TYPE-1")
+};
+
 describe("storage endpoint test", () => {
+  it("should create document when no placeholder object is there", async done => {
+    await request
+      .get("/storage/create")
+      .set("Content-Type", "application/json")
+      .set("Accept", "application/json")
+      .send({
+        document: ropstenDocument
+      })
+      .expect("Content-Type", /json/)
+      .expect(200)
+      .expect(async (err, res) => {
+        if (err) return done(err);
+        expect(res.body).toEqual(thatIsUploadResponse);
+        return remove({ Bucket: config.bucketName, Key: res.body.id });
```

Comment:
what about uploads that succeed but dont look like the expected response? 
e.g someone makes a version update and then the shape of the response changes
very common scenario